packages/client/components/Snackbar.tsx

```diff
@@ -57,6 +60,7 @@ const Snackbar = React.memo(() => {
   const {openPortal, terminatePortal, portal} = usePortal({id: 'snackbar', noClose: true})
   const {location} = useRouter()
   const isMeetingRoute = location.pathname.startsWith('/meet/')
+  const isStandupRoute = isMeetingRoute && !location.pathname.includes('/checkin')
```

Comment:
-2 this will be true even for non-standup routes, like `/meet/i0m8N96why/reflect`, which is a retrospective meeting. to make sure we're on the standup route, you need to check against pattern like `/meet\/.*\/responses/g`